server/api/db/sqldb/db.py

```diff
@@ -5115,6 +5115,7 @@ def enrich_alert(self, session, alert: AlertConfig):
             else mlrun.common.schemas.AlertActiveState.INACTIVE
         )
         alert.count = state.count
+        alert.created = datetime.now(timezone.utc)
```

Comment:
`enrich_alert` is also called in `get_enriched_alert`, and there you don't want to override the created field.
move it to `_create_alert`